packages/client/src/components/inspector/InspectorDataField/Actions.vue

```diff
@@ -129,7 +138,7 @@ function quickEdit(v: unknown, remove: boolean = false) {
       <template #popper>
         <div class="w160px py5px">
           <VueDropdownButton
-            @click="copy(dataType === 'object' ? JSON.stringify(rawValue) : rawValue.toString())"
+            @click="copy(typeof rawValue === 'object' ? JSON.stringify(rawValue) : rawValue.toString())"
```

Comment:
`dataType === 'object'` Cannot correctly diverge the type of rawValue,
Fix to `typeof rawValue === 'object'` to diverge the type.